src/runtime-rs/crates/resource/src/volume/direct_volume.rs

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::sync::Arc;
+
+use anyhow::{anyhow, Context, Result};
+use kata_types::mount::DirectVolumeMountInfo;
+use nix::sys::{stat, stat::SFlag};
+use tokio::sync::RwLock;
+
+use hypervisor::device::device_manager::DeviceManager;
+
+use super::{
+    block_volume, spdk_volume,
+    utils::{
+        get_direct_volume_path, volume_mount_info, KATA_DIRECT_VOLUME_TYPE, KATA_MOUNT_BIND_TYPE,
+        KATA_SPDK_VOLUME_TYPE, KATA_SPOOL_VOLUME_TYPE, KATA_VFIO_VOLUME_TYPE,
+    },
+    vfio_volume, Volume,
+};
+
+pub(crate) fn is_block_volume(m: &oci::Mount) -> Result<bool> {
```

Comment:
From the current design, it is more appropriate to use the `is_direct_volume` then the `is_block_volume`.

Reply:
fixed